### DISCUSSION TITLE
Custom metal side panels

### DISCUSSION THREAD

**Discussion by PaulFreund** [SynthstromAudible/DelugeFirmware#242] [Q&A] [answered]
I want to make custom side panels from metal similar to the main top/bottom housing with and without cutout for the debug connector. @ian-jorgensen it would be very nice if you can give me some insights! I will publish the STEP file once I am done.

Questions I have:
1. What material is used? Is it aluminum AlMg3 or something similar?
2. What is the thickness of the raw metal plates, I measured around 01.55mm so I assume they are 1.5mm?
3. What kind of coating (powder/acrylic paint) is used and if specified what kind of black

Any hints will be helpful :)

Best regards,

Paul

**Comment by ian-jorgensen** [comment, marked as answer]
Zinc plated steel, 1.5, haha "Smooth Matte black" is the only descriptor I have from the company, it's power-coated.

**Reply by PaulFreund** [reply]
Thank you so much for the quick and detailed answer! :)

**Comment by PaulFreund** [comment]
Just to give a little update, since I was not able to easily find the colors I wanted I ended up hand making side panels from an upcycled PCB instead:

![PXL_20230817_214620286](https://github.com/SynthstromAudible/DelugeFirmware/assets/678367/bf56f234-3034-455b-b83c-c1105e6e68cc)

![PXL_20230817_214550578](https://github.com/SynthstromAudible/DelugeFirmware/assets/678367/3584c444-d7fd-439d-9d11-c5615e9aec8e)
